debugging/netcoreapp/prereqManager.ts

```diff
@@ -0,0 +1,93 @@
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { BrowserClient } from './browserClient';
+import { MacNuGetPackageFallbackFolderPath } from './dockerManager';
+import { FileSystemProvider } from './fsProvider';
+import { OSProvider } from './osProvider';
+
+export interface Prerequisite {
+    checkPrerequisite(): Promise<boolean>;
+}
+
+type showErrorMessageFunction = (message: string, ...items: vscode.MessageItem[]) => Thenable<vscode.MessageItem>;
+
+export class DotNetExtensionInstalledPrerequisite implements Prerequisite {
+    constructor(
+        private readonly browserClient: BrowserClient,
+        // tslint:disable-next-line:no-any
+        private readonly getExtension: (extensionId: string) => vscode.Extension<any>,
```

Comment:
any -> unknown

Reply:
Isn't `unknown` only available in TypeScript 3.0?

Reply:
You can merge this in: https://github.com/Microsoft/vscode-docker/pull/528